swap/src/monero/wallet_rpc.rs

```diff
@@ -154,11 +118,126 @@ impl WalletRpc {
         })
     }
 
-    fn tar_path(&self) -> PathBuf {
+    fn archive_path(&self) -> PathBuf {
```

Comment:
🔧 I think this has to distinguish between windows and linux/mac as well - or are you specifying the `zip` will have a name ending in`tar`?

As far as I could see `archive_path` is used in both download functions.

Reply:
I am specifying that the `zip` will have a name ending in `.tar`. I can rename it to `.archive` so that it is less confusing.